docs/git/using.md

```diff
@@ -48,6 +48,7 @@ from Zulip's main repositories.
 projects. With Zulip, because we don't use merge commits, you'll want to avoid
 it. Rather than using `git pull`, which by default is a shortcut for `git fetch
 && git merge FETCH_HEAD` ([docs][gitbook-git-pull]), you should use `git fetch`
+
 and then `git rebase`.
```

Comment:
Unrelated change?

Reply:
Yes, it was unrelated change. It won't happen again. I have reverted it back.